sympy/solvers/ode/single.py

```diff
@@ -852,5 +852,80 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [Eq(lhs, C2 + x), Eq(lhs, C2 - x)]
 
 
+class Lienard(SinglePatternODESolver):
+    r"""
+    The general form of the Lienard ODE:
+
+    .. math :: y''(x) + f(x)*y'(x) + y(x)
+
+    We solve using the fact that all linear second order homogeneous ODEs
+    can be transformed into first order ODEs of Riccati type.
+    Here we solve using the transformation as:
+
+    .. math :: b(x) = y'(x)/y(x)
+
+    .. math :: y''(x) = b'(x)*y(x) + y(x)*b(x)**2
```

Comment:
Is this equation correct?

Reply:
Yes, for `y'(x) = b(x)*y(x)` => On differentiating w.r.t. x, we get,
`y''(x) = b'(x)*y(x) + y'(x)*b(x) = b'(x)*y(x) + y(x)*b(x)**2`, substituing for `y'(x)`.
I skipped a step in the docstring.